spec/features/actors_feature_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "rails_helper"
+
+RSpec.describe "Actors", type: :feature do
+
+  before do
```

Comment:
I want to be sure there's enough data for accurate testing (i.e. multiple records of each) which, in this case, would seriously pollute the tests.  I recognize the students won't know how the data is created, but by using code in the tests to read the data (e.g. `Actor.all` or `Actor.last`), I figure the mystery guest problem is less of a factor.